### PR TITLE
MSTest.Sdk prefer global.json instead of local version

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

**Review by MarcoRossignoli — APPROVED** [review]

File: template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/Playwright-MSTest-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -72,6 +72,20 @@
   ],
   "defaultName": "TestProject1",
   "postActions": [
+    {
+      "description": "Add MSTest.Sdk entry to global.json",
+      "manualInstructions": [ { "text": "Add a new property 'MSTest.Sdk' under 'msbuild-sdks' section of the 'global.json' file." }],
+      "actionId": "695A3659-EB40-4FF5-A6A6-C9C4E629FCB0",
+      "args": {
+        "jsonFileName": ".\\global.json",
```
Comment (ericstj) [inline template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/Playwright-MSTest-CSharp/.template.config/template.json:80]: I wonder if this syntax works on Linux or if the forward slash will be treated as part of the filename?

**Comment by Evangelink** [comment]
Why do you think it would be easier?

**Comment by Evangelink** [comment]
I am sorry @Varorbc but I don't see any explanation of why this syntax is easier from the aspire discussion you pointed.

**Comment by Evangelink** [comment]
I have done some tests and they have the same issue of updates with NuGet as we have and their SDK is different than ours as they don't import the default SDK, probably because they don't need to ensure the correct ordering of insertions.

**Comment by christiannagel** [comment]
> In the .NET ecosystem, I prefer using SDKs with the Microsoft prefix, just like how Aspire is used.

@Varorbc The Aspire SDK has the name `Aspire.AppHost.Sdk`, to `Microsoft` prefix

**Pull request merged**